tests/integration/ceph_multi_cluster_test.go

```diff
@@ -79,6 +81,7 @@ func (mrc *MultiClusterDeploySuite) SetupSuite() {
 
 	mrc.op, mrc.k8sh = NewMCTestOperations(mrc.T, mrc.namespace1, mrc.namespace2)
 	mrc.testClient = clients.CreateTestClient(mrc.k8sh, mrc.op.installer.Manifests)
+	mrc.createOSD(sys.DiskType)
```

Comment:
Could we split up the test a bit differently?
- Here in the `SetupSuite()` create one type of OSD 
- In a new test method, remove the first OSD and create the other types of OSDs. At the end of the test, one osd would still exist in case other OSDs are needed.

The way you have it would work as long as the OSD setup passes, but if any of the other types fails, it would block all other tests in the suite from running. So it is preferred to break it up into the separate test if possible.

Reply:
How about adding the new test suite that is dedicated to OSD on PVC test? It can prevent blocking other tests that don't correspond to OSD on PVC without introducing In addition, it makes the purpose of the multi-cluster test suites clearer.

Reply:
A new suite is also a possibility. We just really need to control what suites we are adding since each one can easily add several minutes to the time to integration tests. I'm ok with a new test suite in this case.